pkg/innerring/processors/netmap/process_peers.go

```diff
@@ -62,26 +61,9 @@ func (np *Processor) processAddPeer(ev netmapEvent.AddPeer) {
 		np.log.Info("approving network map candidate",
 			zap.String("key", keyString))
 
-		prm := netmapclient.AddPeerPrm{}
-		prm.SetNodeInfo(nodeInfo)
-
-		// In notary environments we call AddPeerIR method instead of AddPeer.
-		// It differs from AddPeer only by name, so we can do this in the same form.
-		// See https://github.com/nspcc-dev/neofs-contract/issues/154.
-		const methodAddPeerNotary = "addPeerIR"
-
-		// create new notary request with the original nonce
-		err = np.netmapClient.Morph().NotaryInvoke(
-			np.netmapClient.ContractAddress(),
-			0,
-			originalRequest.MainTransaction.Nonce,
-			nil,
-			methodAddPeerNotary,
-			nodeInfoBinary,
-		)
-
+		err = np.netmapClient.Morph().NotarySignAndInvokeTX(ev.NotaryRequest().MainTransaction)
```

Comment:
btw can we use `originalRequest` here? code will reflect "original request" processing more clear

Reply:
i do not see any api about signing notary requests in `neo-go`. it is always TXs

Reply:
i mean we have https://github.com/nspcc-dev/neofs-node/blob/de84b12e02dc6fe5bc564928d74398b2a32e26d6/pkg/innerring/processors/netmap/process_peers.go#L21-L22

they are the same as `ev.NotaryRequest()` right? seems like that. If so, it's worth to pass `tx` or `originalRequest.MainTransaction` them as method arg